### ISSUE TITLE
Problems with special characters like ü

### ISSUE THREAD

**Issue by dawiinci** [IndigoDomotics/indigo-pushover#26]
```
   Pushover Error                  Error in plugin execution ExecuteAction:

Traceback (most recent call last):
  File "plugin.py", line 119, in send
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 10: ordinal not in range(128)

```

I tried to send a value as a title in a message containing "Büro". This gives an error but sends the message regardless.

%%v:VAR_ID%%

**Comment by jcs** [comment]
All of the `self.logger` calls are passing unicode strings and `msgTitle` and `msgBody` are already encoded to UTF-8, but from this error message, the `logger` object is only taking ASCII input.  I think this is an upstream problem and the Indigo Python framework needs to setup a logger that takes UTF-8 input.

**Comment by FlyingDiver** [comment]
Try this version and see if special characters work:

https://github.com/IndigoDomotics/indigo-pushover/releases/tag/1.5.4

**Comment by indigodomo** [comment]
> All of the `self.logger` calls are passing unicode strings and `msgTitle` and `msgBody` are already encoded to UTF-8, but from this error message, the `logger` object is only taking ASCII input. I think this is an upstream problem and the Indigo Python framework needs to setup a logger that takes UTF-8 input.

I don't believe this is true. Joe, what was the correction?

**Comment by FlyingDiver** [comment]
Changed all string substitution and concatenation to use .format()

**Comment by dawiinci** [comment]
Problem ist unfortunately NOT fixed. 
The same error appears, but the message is sent out like before.

**Comment by FlyingDiver** [comment]
@dawiinci - what line in 1.5.4 is the error on?

**Comment by dawiinci** [comment]
It was on line  121:

`Traceback (most recent call last):
  File "plugin.py", line 121, in send
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 20: ordinal not in range(128)`

**Comment by FlyingDiver** [comment]
Please turn debugging on in the plugin prefs and do the send again, then post the log segment.

**Comment by dawiinci** [comment]
There is no additional content shown with debug. The error message is exactly the same as in 1.5.3.

**Comment by FlyingDiver** [comment]
Hmm.  With debug on, the plugin should be logging the content of the fields it's preparing to send.  Also, the line numbers should have changed.  I want to make sure I know exactly what line is throwing the error.

**Comment by FlyingDiver** [comment]
Fixed in (new) 1.5.4.  I reused the release number since it was never officially released.

**Comment by dawiinci** [comment]
Thank you, it works in 1.5.4.
